account_avatax/models/account_move.py

```diff
@@ -407,11 +421,21 @@ def create(self, vals):
             record.avatax_compute_taxes()
         return record
 
+    def action_reverse(self):
+        action = super().action_reverse()
+        avatax_tax_type = self.invoice_line_ids.filtered(lambda t: t.avatax_tax_type)
+        action["context"] = {
+            "default_avatax_amt_line_override": self.avatax_amt_line_override,
+            "hide_override": 1 if avatax_tax_type else 0,
+        }
```

Comment:
I'm not sure it is safe to override the action context :-/
```suggestion
        action["context"] = action.get("context", {}).update(
            {
                "default_avatax_amt_line_override": self.avatax_amt_line_override,
                "hide_override": 1 if avatax_tax_type else 0,
            }
        )
```